wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -1091,6 +1091,11 @@ extension ZMUserSession: SyncAgentDelegate {
                 context: notificationContext
             ).post()
 
+            guard !isRecovering else {
```

Comment:
added this as I don't think we should do any extra task when recovering from quick sync